File: app/src/main/java/android/asiantech/vn/springfinalmusic/loading/LoadingActivity.java
package android.asiantech.vn.springfinalmusic.loading;

import android.app.Activity;
import android.asiantech.vn.springfinalmusic.R;
import android.asiantech.vn.springfinalmusic.home.HomeActivity;
import android.asiantech.vn.springfinalmusic.service.MusicService;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.Button;

public class LoadingActivity extends Activity {
    Button button;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.test);
        button=findViewById(R.id.btnTest);
    }

    @Override
    protected void onStart() {
        super.onStart();
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(LoadingActivity.this, MusicService.class);
                startService(intent);
            }
        });
        //activeHome();
    }

    private void activeHome() {
        Intent intentHome = new Intent(this, HomeActivity.class);
        startActivity(intentHome);
    }
}

File: app/src/main/java/android/asiantech/vn/springfinalmusic/loading/LoadingActivity.kt
package android.asiantech.vn.springfinalmusic.loading

import android.app.Activity
import android.asiantech.vn.springfinalmusic.R
import android.asiantech.vn.springfinalmusic.home.HomeActivity
import android.asiantech.vn.springfinalmusic.manager.ResourcesManager
import android.content.Intent
import android.os.Bundle
import android.os.Handler

class LoadingActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_loading)
        ResourcesManager.getInstance().loadResources(this)
    }

    override fun onStart() {
        super.onStart()
        val handler = Handler()
        handler.postDelayed({
            val intent = Intent(this@LoadingActivity, HomeActivity::class.java)
            startActivity(intent)
            finish()
        }, 2000)
    }
}
